DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,13 @@ public function getConfigTreeBuilder()
         $rootNode->setBuilder(new AllowExtraPropertiesNodeBuilder());
 
         $rootNode->children()
+            ->arrayNode('allowedParamConverters')
+                ->prototype('array')
+                    ->children()
+                        ->scalarNode('name')->isRequired()->end()
```

Comment:
How about simply have a list here? Are we expecting any additional information to be needed here?
```yaml
allowedParamConverters:
  - "kfz24.commons.json_schema.request_body"
```